client/my-sites/domains/domain-management/name-servers/wpcom-nameservers-toggle.jsx

```diff
@@ -18,20 +18,21 @@ class NameserversToggle extends PureComponent {
 	};
 
 	render() {
+		const toggleTitle = this.props.translate( 'Use WordPress.com Name Servers' );
 		return (
 			<Card compact className="name-servers__dns">
-				<span className="name-servers__title">
-					{ this.props.translate( 'Use WordPress.com Name Servers' ) }
-				</span>
-
 				<form className="name-servers__toggle">
 					<ToggleControl
 						id="wp-nameservers"
 						name="wp-nameservers"
 						onChange={ this.handleToggle }
 						checked={ this.props.enabled }
 						value="active"
+						label={ toggleTitle }
+						className="name-servers__title"
+						onClick={ this.handleToggle }
 					/>
+					<ScreenReaderText>{ toggleTitle }</ScreenReaderText>
```

Comment:
Is the `<ScreenReaderText />` necessary now that we're using the built-in label from the `ToggleControl` component? Can we just remove it if that's the case?

Reply:
I actually did not know that the ToggleControl Label would suffice, I have removed it now. I also proceeded to test with the Mac voiceover and it works great! besides no console error. You are teaching me `attention to detail`. Thank you !

Reply:
Nice! And yep, I confirmed it's working great now 🎉